SuperCalculatorConverter/SuperCalculatorConverter/ProcessOutput.cpp

```diff
@@ -0,0 +1,244 @@
+#include "ProcessOutput.h"
+#include "ShuntingAlgorithm.h"
+#include "Converter.h"
+
+using namespace InputOutput;
+using namespace ShuntingAlgorithmCalculator;
+using namespace Conversions;
+
+Converter converter;
+
+std::string ProcessOutput::InputOutput(std::string input) {
+	if (equationInput.empty() && !isdigit(input.back())) {
+		if (input.back() == '.') {
+			equationInput = "0" + input;
```

Comment:
couldn't equationInput be some kind of string stream? all this concatenation is a little bit yikes.